ddtrace/internal/gitmetadata.py

```diff
@@ -100,8 +105,8 @@ def get_git_tags():
         config = GitMetadataConfig()
 
         if config.enabled:
-            repository_url, commit_sha = _get_tags_from_env(config)
-            log.debug("git tags from env: %s %s", repository_url, commit_sha)
+            repository_url, commit_sha, main_package = _get_tags_from_env(config)
```

Comment:
IMO if we are extracting the main package here from the config, it would slightly help the readability to change _get_tags_from_package to accept directly the package name, and have the logic just below:

```python
if main_package and (not repository_url or not commit_sha):
    # trying to extract repo URL and/or commit sha from the main package
    pkg_repository_url, pkg_commit_sha = _get_tags_from_package(main_package)
```